io/net/virt-net/network_virtualization.py

```diff
@@ -305,7 +305,7 @@ def test_disable_enable_dev(self):
         self.is_disabled_enabled_dev('1')
         self.disable_enable_dev('e')
         self.is_disabled_enabled_dev('0')
-
+        time.sleep(60)
```

Comment:
ah  somehow i did not like adding sleep time in Test can we think to avoid this

Reply:
added wait function instead sleep